src/RuntimeStoreVerifier/RuntimeStoreVerifierCommand.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.CommandLineUtils;
+using WinTrustSources;
+
+namespace RuntimeStoreVerifier
+{
+    public class RuntimeStoreVerifierCommand
+    {
+        internal static int Execute(
+            CommandOption exclusionFileOption,
+            CommandOption verboseOption,
+            CommandArgument pathArgument)
+        {
+            var exclusions = exclusionFileOption.HasValue() ? File.ReadAllLines(exclusionFileOption.Value()) : null;
```

Comment:
This can still be an MSBuild task, regardless of where it lives (repotask or here). To make it an MSBuild task, we just need to make a few tweaks to the project and add a UsingTask somewhere.